Tests/PSGetFindModule.Tests.ps1

```diff
@@ -24,12 +24,13 @@ function SuiteSetup {
     $script:MyDocumentsModulesPath = Get-CurrentUserModulesPath
     $script:PSGetLocalAppDataPath = Get-PSGetLocalAppDataPath
     $script:DscTestModule = "DscTestModule"
+    $script:PrereleaseTestModule = "TestPackage"
```

Comment:
I agree.  When I push the prerelease stuff to INT, I'll create a PrereleaseTestModule that the tests will run against.  For now, these tests are running against my local gallery, and it was easier to reuse an existing test module.  I will leave this open until the INT work is done.

---
In reply to: [132585600](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132585600) [](ancestors = 132585600,126798793)